$NetBSD: patch-ab,v 1.13 2010/10/12 19:50:35 joerg Exp $

Fix unportable test construct.
LD_PRELOAD is supported on DragonFly.

--- configure.orig	2010-09-06 16:16:46.000000000 +0000
+++ configure
@@ -7392,6 +7392,9 @@ sparc*-*solaris*)
   fi
   rm -rf conftest*
   ;;
+*-*qnx6*)
+  PTHREAD_LIBS="-lc"
+  ;;
 esac
 
 need_locks="$enable_libtool_lock"
@@ -21447,7 +21450,7 @@ xlib_xcb_NONPKGCONFIG_LIBS="$ac_env_xlib
 xlib_xcb_NONPKGCONFIG_EXTRA_LIBS="$ac_env_xlib_xcb_NONPKGCONFIG_EXTRA_LIBS_value"
 
 
-  if test "x$use_xcb" == "xyes" -a "x$use_xlib" == "xyes"; then
+  if test "x$use_xcb" = "xyes" -a "x$use_xlib" = "xyes"; then
     xlib_xcb_REQUIRES="x11-xcb"
 
 pkg_failed=no
@@ -21797,7 +21800,7 @@ xcb_shm_NONPKGCONFIG_LIBS="$ac_env_xcb_s
 xcb_shm_NONPKGCONFIG_EXTRA_LIBS="$ac_env_xcb_shm_NONPKGCONFIG_EXTRA_LIBS_value"
 
 
-  if test "x$use_xcb" == "xyes"; then
+  if test "x$use_xcb" = "xyes"; then
       xcb_shm_REQUIRES="xcb-shm"
 
 pkg_failed=no
@@ -25203,7 +25206,7 @@ drm_xr_NONPKGCONFIG_LIBS="$ac_env_drm_xr
 drm_xr_NONPKGCONFIG_EXTRA_LIBS="$ac_env_drm_xr_NONPKGCONFIG_EXTRA_LIBS_value"
 
 
-  if test "x$use_drm" == "xyes"; then
+  if test "x$use_drm" = "xyes"; then
       drm_xr_REQUIRES="xorg-server >= 1.6 xproto xextproto >= 7.0.99.1 renderproto x11"
 
 pkg_failed=no
@@ -25845,7 +25848,7 @@ xcb_drm_NONPKGCONFIG_LIBS="$ac_env_xcb_d
 xcb_drm_NONPKGCONFIG_EXTRA_LIBS="$ac_env_xcb_drm_NONPKGCONFIG_EXTRA_LIBS_value"
 
 
-  if test "x$use_xcb" == "xyes" -a "x$use_drm" == "xyes"; then
+  if test "x$use_xcb" = "xyes" -a "x$use_drm" = "xyes"; then
       xcb_drm_REQUIRES="xcb-dri2"
 
 pkg_failed=no
@@ -33603,7 +33606,7 @@ _ACEOF
 
 
 case $host in
-*-linux*|*-*bsd*|*-solaris*|*-*-darwin*)
+*-linux*|*-*bsd*|*-solaris*|*-*-darwin*|*-dragonfly*|*-*qnx6*)
 	have_ld_preload="yes"
 	;;
 *)
